sstables/sstables.hh

```diff
@@ -463,6 +463,32 @@ public:
         });
     }
     future<> close_files();
+
+    /* Returns a lower-bound for the set of `position_in_partition`s appearing in the sstable across all partitions.
```

Comment:
Unrelated change.

Reply:
I'm assuming you'd like it to be moved to a separate commit?

Reply:
Yes